pkg/apiserver/utils/auth.go

```diff
@@ -32,6 +32,9 @@ import (
 // KubeVelaProjectGroupPrefix the prefix kubevela project
 const KubeVelaProjectGroupPrefix = "kubevela:project:"
 
+// KubeVelaAdminGroupPrefix the prefix kubevela admin
+const KubeVelaAdminGroupPrefix = "kubevela:admin:"
```

Comment:
The admin is for velaux? I suggest using an alternative name like "kubevela:ux:admin"

Reply:
I think the name is a little redundant.

Reply:
"kubevela:admin" is confusing for CLI users.

Reply:
"kubevela:project" since there is no so-called "project" in CLI while "admin" is widely used.